src/elements/content-preview/PreviewNavigation.js

```diff
@@ -14,11 +17,19 @@ import messages from '../common/messages';
 type Props = {
     collection: Array<string | BoxItem>,
     currentIndex: number,
+    history: RouterHistory,
     onNavigateLeft: Function,
     onNavigateRight: Function,
 } & InjectIntlProvidedProps;
```

Comment:
Observation: Since InjectIntlProvidedProps is not imported in this file then it resolves to an `any(implicit)` type.  When it is imported, flow resolves the type properly which causes flow errors for the history prop in the ContentPreview component.

Reply:
We need to fix this in this PR, the current `any` type is just hiding the issue